auth-web/src/components/auth/manage-business/AffiliationAction.vue

```diff
@@ -424,9 +450,7 @@ export default defineComponent({
     const handleApprovedNameRequest = (item: Business, nrRequestActionCd: NrRequestActionCodes): void => {
       switch (nrRequestActionCd) {
         case NrRequestActionCodes.AMALGAMATE:
-          // For now, go to COLIN with external link + icon + matching hover text
-          // Future gotoAmagamate
-          goToCorpOnline()
+          goToAmalgamate(item)
```

Comment:
The user will still go to Corp Online for non-supported entity types, right?

Reply:
Yes, Sev. For example right now, Everything except benefit companies will go to COLIN since the FF is "BEN".